onnx/defs/rnn/defs.cc

```diff
@@ -82,7 +110,7 @@ Each parameter matrix is linearly appended after the previous parameter matrix
 without padding.
 
 The order of matrixes `{K, L, D, R, N, C}` is defined as:
```

Comment:
you'd need to update the comment here too - for example change 'relu' / 'tanh' to 'simple' and update layout order

Reply:
Thanks. Done.
Regarding the layout order in the weights input, I don't have a complete picture. Take a simple example, for LSTM, it is not completely clear to me how we pack Wi, Wf, Wo, and Wc in the weight tensor. Is it [4*hidden_size, input_size] or [hidden_size, 4*input_size]?  Perhaps I missed something obvious.

Reply:
It's actually more complicated as we're trying to pack several layers of e.g. LSTM together. The first layer would have different input size from later ones, the input gates might not have inputs at all, etc. Right now I just concatenated all the weight matrices together. It's not pretty but solves the problem of too many inputs.

Thus for peep_hole connection I'd just modify the message to say that there are more weight matrices in the internal order (R being "input-hidden", "hidden-hidden" and "peephole").

Reply:
@dzhulgakov As we discussed, I will try to update this PR to have 4 RNN ops: SimpleRNN, GRU, LSTM, and BirectionalLSTM. I think that this would make things simpler.

Reply:
Yes, adding separate ops would make it simpler. The only problem is that stacked nature - for example the input-hidden matrix has different sizes for the first and subsequent layers.